lib/caseflow/service.rb

```diff
@@ -76,8 +81,8 @@ def healthcheck
 
     private
 
-    def request_headers(additional_headers = {})
-      DEFAULT_HEADERS.merge(additional_headers)
+    def authorized_perform(method, path, params, additional_headers = nil, options = nil)
+      perform(method, path, params, DEFAULT_HEADERS.merge(additional_headers || {}), options)
```

Comment:
i'd prefer this to match the arity of `perform` (going against code climate's suggestion)

Reply:
I don't have an issue with this if we are the only ones really using this lib for now.